test/sampler/test_sampler.py

```diff
@@ -53,6 +59,8 @@
 hi = nk.hilbert.Spin(s=0.5, N=g.n_nodes)
 ha = nk.operator.Ising(hilbert=hi, graph=g, h=1.0)
 move_op = sum([nk.operator.spin.sigmax(hi, i) for i in range(hi.size)])
+move_weights = np.array([2 ** i for i in range(hi.size)])
```

Comment:
oh sorry, I see... you are putting this kind of weights, still I would go for random ones, less chance for error

Reply:
Yeah, I went for random ones